### PR TITLE
Add username to admin jobs payload

### PR DESCRIPTION
**Pull request by johnbradley**
# Changes to `/api/admin/jobs/`
Removes user_id and replaces it with user containing id and username.
This is to allow transferring the completed project to the user once the
files have been uploaded.

Old format for a single job:
```
{
    "id": 1,
    ....
    "user_id": 4,
    ...
}
```

New format for a single job:
```
{
    "id": 1,
    ...
    "user": {
        "id": 2,
        "username": "dcl9@duke.edu"
    },
    ...
}
```

### CODE REVIEW

**Review by dleehr — COMMENTED** [review]
This adds a serializer to embed a user object inside each job from `/api/admin/jobs`. That appears to work for the stated goal (getting the user's name) for the admin client. How does that impact other uses of the endpoint - creating or updating jobs? Would it create a new user if the JSON was correct? is that something we want to be able to do?

Another option - what about leaving the user id in the serializer, but adding a **read-only**`/api/admin/users` endpoint.

**Comment by johnbradley** [comment]
User is a read only field of /api/admin/jobs/ so it would not create a new user.

File: data/serializers.py [hunk 1]
```diff
@@ -69,16 +69,21 @@ class Meta:
...
+    username = serializers.CharField()
+
+
 class AdminJobSerializer(serializers.ModelSerializer):
     workflow_version = WorkflowVersionSerializer(required=False)
     output_dir = JobOutputDirSerializer(required=False, read_only=True)
     vm_project_name = serializers.CharField(required=False)
-    user_id = serializers.IntegerField(required=False)
     name = serializers.CharField(required=False)
+    user = UserSerializer(read_only=True)
```
Comment (dleehr) [inline data/serializers.py:82]: I see

**Comment by dleehr** [comment]
👍

**Pull request merged**